nrf_dm/doc/CHANGELOG.rst

```diff
@@ -12,6 +12,16 @@ All the notable changes to this project are documented in this file.
 main
 ****
 
+Changes
+=======
+* Improved the resilience to interference during synchronization.
+* Improved the quality of access addresses used internally in the library.
+* ``access_address`` was renamed to ``rng_seed`` in :c:type:`nrf_dm_config_t`.
```

Comment:
```suggestion
* Renamed ``access_address`` to ``rng_seed`` in :c:type:`nrf_dm_config_t`.
```
Makes us sound more active, instead of something just happening.